tasks/subscription-manager.yml

```diff
@@ -33,3 +33,16 @@
     server_insecure: "{{ rhc_server.insecure | d(omit) }}"
     auto_attach: "{{ true
       if ( rhc_state | d('present') == 'present' ) else omit }}"
+
+- name: Configure subscribed system
+  block:
+    - name: Configure repositories
+      rhsm_repository:
+        name: "{{ item.name }}"
+        state: "{{ item.state }}"
+        purge: no
+      loop: "{{ rhc_repositories }}"
+      when:
+        - ( rhc_repositories | d([]) ) | length > 0
+  when:
```

Comment:
The new ansible recommendation is to have the `block/rescue/always` be the last keywords in the task - any `when` or `vars` should come after `name` and before `block`  - ansible-lint 6.x will flag this